src/Forms/FormScaffolder.php

```diff
@@ -39,17 +46,27 @@ class FormScaffolder
      */
     public $restrictFields;
 
+    /**
+     * Numeric array of field names and has_one relations to explicitly not scaffold.
+     */
+    public array $ignoreFields = [];
+
     /**
      * @var array $fieldClasses Optional mapping of fieldnames to subclasses of {@link FormField}.
      * By default the scaffolder will determine the field instance by {@link DBField::scaffoldFormField()}.
      */
     public $fieldClasses;
 
     /**
-     * @var boolean $includeRelations Include has_one, has_many and many_many relations
+     * @var boolean $includeRelations Include has_many and many_many relations
```

Comment:
This was intentionally changed to not include has_one in https://github.com/creative-commoners/silverstripe-framework/commit/524d7a90117d447bc7705066500b1637aa75f0e0